### PR TITLE
[2.8] Remove 2.6 versionaddeds as version reached eom

### PR DESCRIPTION
**Pull request by wouterj**
2.6 has reached eom today, so 2.8 should no longer contain the versionadded directives for 2.6 according to our upgrade process.

| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | 2.8+ |
| Fixed tickets | - |

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1:

File: reference/twig_reference.rst [hunk 1]
```diff
@@ -721,10 +715,6 @@ The available attributes are:
 * ``app.debug``
 * ``app.security``
 
-.. versionadded:: 2.6
-     The ``app.security`` global is deprecated as of 2.6. The user is already available
-     as ``app.user`` and ``is_granted()`` is registered as function.
```
Comment (stof) [inline reference/twig_reference.rst]: Removing this one is wrong. The explanation saying it is deprecated is necessary on 2.8

File: cookbook/controller/service.rst [hunk 2]
```diff
@@ -122,10 +122,6 @@ the route ``_controller`` value:
     defined as a service. See the `FrameworkExtraBundle documentation`_ for
     details.
 
-.. versionadded:: 2.6
-    If your controller service implements the ``__invoke`` method, you can simply refer to the service id
-    (``app.hello_controller``).
```
Comment (stof) [inline cookbook/controller/service.rst]: this information should not be lost

File: components/console/helpers/progressbar.rst [hunk 3]
```diff
@@ -40,22 +40,11 @@ Instead of advancing the bar by a number of steps (with the
...
     Prior to version 2.6, the progress bar only works if your platform
     supports ANSI codes; on other platforms, no output is generated.
 
-.. versionadded:: 2.6
-    If your platform doesn't support ANSI codes, updates to the progress
-    bar are added as new lines. To prevent the output from being flooded,
-    adjust the
-    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::setRedrawFrequency`
-    accordingly. By default, when using a ``max``, the redraw frequency
-    is set to *10%* of your ``max``.
```
Comment (stof) [inline components/console/helpers/progressbar.rst]: this info should not be lost

File: book/templating.rst [hunk 4]
```diff
@@ -1222,12 +1222,6 @@ automatically:
             <p>Application Environment: <?php echo $app->getEnvironment() ?></p>
         <?php endif ?>
 
-.. versionadded:: 2.6
-    The global ``app.security`` variable (or the ``$app->getSecurity()``
-    method in PHP templates) is deprecated as of Symfony 2.6. Use ``app.user`` 
-    (``$app->getUser()``) and ``is_granted()`` (``$view['security']->isGranted()``)
-    instead.
```
Comment (stof) [inline book/templating.rst]: The fact that ``app.security`` is deprecated should not be lost

Comment (WouterJ) [inline book/templating.rst]: `app.security` isn't used here, so it doesn't make sense to have this tip box here.

**Comment by WouterJ** [comment]
Thanks for the review @stof! Instead of adding the versionadded directives with important information back, I've created a PR to an older branch to move this information out of the versionadded: https://github.com/symfony/symfony-docs/pull/5641

**Comment by xabbuh** [comment]
@WouterJ This needs a rebase.

**Comment by WouterJ** [comment]
Rebased

**Comment by xabbuh** [comment]
Thank you Wouter.

**Pull request merged**